cmake/SetupBlaze.cmake

```diff
@@ -19,24 +19,32 @@ set_property(TARGET Blaze PROPERTY
   INTERFACE_INCLUDE_DIRECTORIES ${BLAZE_INCLUDE_DIR})
 set_property(TARGET Blaze PROPERTY
   INTERFACE_LINK_LIBRARIES Lapack)
+target_link_libraries(
+  Blaze
+  INTERFACE
+  Blas
+  GSL::gsl # for BLAS header
+  Lapack
+  )
 
 target_compile_definitions(Blaze
   INTERFACE
-  # Override SMP configurations
+  # Enable external BLAS kernels
+  BLAZE_BLAS_MODE=1
+  # Set default matrix storage order to column-major
+  BLAZE_DEFAULT_STORAGE_ORDER=blaze::columnMajor
+  # Disable SMP parallelization
   BLAZE_USE_SHARED_MEMORY_PARALLELIZATION=0
-  BLAZE_OPENMP_PARALLEL_MODE=0
```

Comment:
Please keep all parallelization things explicitly disabled. We've had issues with blas libraries doing multithreading on us in the past.

Reply:
The `BLAZE_USE_SHARED_MEMORY_PARALLELIZATION` disables SMP parallelization altogether, see: https://bitbucket.org/blaze-lib/blaze/wiki/Serial%20Execution#!option-3-deactivation-of-parallel-execution